be/src/olap/rowset/beta_rowset_writer.cpp

```
@@ -664,6 +739,13 @@ Status BaseBetaRowsetWriter::_build_rowset_meta(RowsetMeta* rowset_meta, bool ch
     // TODO write zonemap to meta
     rowset_meta->set_empty((num_rows_written + _num_rows_written) == 0);
     rowset_meta->set_creation_time(time(nullptr));
+
+    if (auto seg_file_size = _seg_files.segments_file_size(); !seg_file_size.has_value()) {
+        LOG(ERROR) << seg_file_size.error();
```

Should we return an error status?